kubectl-pg/cmd/delete.go

```diff
@@ -0,0 +1,114 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/zalando/postgres-operator/pkg/apis/acid.zalan.do/v1"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	"io/ioutil"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/scheme"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+)
+
+// deleteCmd represents kubectl pg delete.
+var deleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Deletes postgresql object by object-name/manifest file",
```

Comment:
we normally speak of postgresql manifests, not objects

Reply:
Here the delete works in two ways either by specifying postgres manifest file or by using cluster name.
```kubectl pg delete -f minimal-cluster.yaml```
```kubectl pg delete cluster01```
I will change it to object-name -> cluster-name